Calculator_Project/Source.cpp

```diff
@@ -0,0 +1,251 @@
+#include <iostream>
+#include <vector>
+#include <iterator>
+#include <string>
+
+template <typename T>
+void vec_print(const std::vector<T>& container) {   //Print vector, entry by entry
+    for (const T equation : container)              //Found online, modified to generic type
+        std::cout << equation << ' ';
+    std::cout << std::endl;
+}
+
+bool char_is_num(const char c) {        //Return true if character c is a numeral, false if not
+    if ((c == '0') or (c == '1') or (c == '2') or (c == '3') or (c == '4') or (c == '5') or (c == '6') or (c == '7') or (c == '8') or (c == '9')) {

```

Comment:
Can replace this with ` if (c >= '0' && c <= '9' `, because 0 through 9 are sequentially ordered in ASCII.